schema/core-im/core-im-source.yaml

```diff
@@ -80,7 +80,7 @@ $defs:
           generated.
       contributions:
         type: array
-        ordered: true
```

Comment:
I had thought contributions would be ordered chronologically–not so?

Reply:
based on our discussion on this, I will revert it. No harm in making the ordering true and will be useful for some use cases.